src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import duke.tasks.Task;
```

Comment:
I noticed that your Duke.java file is outside the `duke` folder. You might want to move it into that folder such that all your code is under the folder. You could use the `move class` feature under refactor to achieve it.